src/spec.h

```diff
@@ -314,6 +314,7 @@ typedef struct IndexSpec {
   bool cascadeDelete;             // (deprecated) remove keys when removing spec. used by temporary index
   bool monitorDocumentExpiration;
   bool monitorFieldExpiration;
+  bool isDuplicate;               // Marks that this index is a duplicate of an existing one
```

Comment:
This can't really happen other than during loading from a persistence file (rdb, aof), so why have it as a field?
I think it would be better not to do have it.

Reply:
We use this field in `IndexSpec_RemoveFromGlobals`

Reply:
It can be an argument to the function, better than an almost unused field IMO.
If we want a fast fix that does not seek to be optimal, but similar to what we had before I guess we can go forward with it, but I do think it is not ideal.